aucoalesce/coalesce.go

```diff
@@ -34,6 +34,13 @@ import (
 // package does not have a constant defined for this.
 const modeBlockDevice = 060000
 
+// ECSEvent contains ECS-specific categorization fields
+type ECSEvent struct {
```

Comment:
**Non blocking**
Would event.kind be held somewhere else?  or maybe the caller of the library is responsible for that?

What do you think about having something about ECS categorization in the struct type?  That might provide clues to the type of fields in the struct.

Reply:
my initial thought is that it could be in the caller--namely because I'm pretty sure we're working only with `event` kinds--but then again would "anomaly" events actually be `alert` kinds? If so, or if we just want to be explicit, I can add `kind` in a follow-up PR.

For the second question, I'm not entirely sure I follow--do you meaning adding constant enums for the categorization values themselves? If so I'd almost prefer to do that in the ECS go generated code from the ECS repo itself (which ideally we'd be using here too--but we still need to add tags and formalize on things like `omitempty`, pointer usage, etc. before doing that).

Reply:
I'm good with it being the callers responsibility, I just wanted to double check that we weren't missing one of the ECS categorization fields.

I was thinking the name of the type might be better as ECSCategorization or ECSCat or something like that, since it really just holds the categorization info, nothing else about ECS.

Enums would be awesome and I agree it would be good to do that work once in ECS repo.